packages/components/src/Divider.astro

```diff
@@ -0,0 +1,46 @@
+---
+import { classMerge } from "@gemini-ui-astro/core";
+
+export interface DividerProps {
+  class?: string;
+  thickness?: string;
+  orientation?: "horizontal" | "vertical";
+  color?: string;
+  mt?: string;
+  mb?: string;
+  ml?: string;
+  mr?: string;
+}
+
+const {
+  class: className,
+  thickness = "1px",
+  orientation = "horizontal",
+  color = "#323232",
+  mt = 0,
+  mb = 0,
+  ml = 0,
+  mr = 0,
+} = Astro.props;
+---
+
+<hr class={classMerge("gx-divider", className, `gx-divider-${orientation}`)} />
+
+<style lang="scss" define:vars={{ thickness, color, mt, mb, ml, mr }}>
+  .gx-divider {
+    border: 0;
+    margin: 0;
+    border-top: var(--thickness) solid var(--color);
```

Comment:
Should we be using `top`, `bottom`, `left`, `right` or the logical properties `block-start`, `block-end`, `inline-start`, `inline-end`?

Typically using logical properties would be best practice to better support languages which read in different orientations (e.g. up/down or right to left). However, users of the library probably would expect properties like `mt` to translate to `top`, not the equivalent logical property.

Thoughts? 🤔